srv/components/ha/haproxy/files/haproxy.cfg

```diff
@@ -73,27 +73,24 @@ defaults
 #---------------------------------------------------------------------
 # main frontend which proxys to the backends
 #---------------------------------------------------------------------
-# frontend main
-#     bind *:5000
-#     acl url_static       path_beg       -i /static /images /javascript /stylesheets
-#     acl url_static       path_end       -i .jpg .gif .png .css .js
+frontend main
+    bind 127.0.0.3:28000
```

Comment:
Can we put IP/Port in some common config file (where we define Provisioner specific port defns)

Reply:
Will take it up in a later PR